components/logging/org.wso2.carbon.logging.service/src/main/java/org/wso2/carbon/logging/service/util/LoggingUtil.java

```diff
@@ -442,7 +442,7 @@ public static void restoreDefaults() throws Exception {
         } catch (FileNotFoundException e) {
             String msg = "Cannot restore default logging configuration."
                     + " log4j.properties file not found in the classpath";
-            throw new Exception(msg, e);
+            throw new LogViewerException(msg, e);
```

Comment:
Is there any specific reason for creating a different variable for the exception message?

Reply:
@nishadi this is the pattern we have followed in most of the classes.